src/plugins/data_source_management/public/types.ts

```diff
@@ -121,7 +121,7 @@ export const sigV4CredentialField = {
   region: '',
   accessKey: '',
   secretKey: '',
-  service: '',
+  service: SigV4ServiceName.OpenSearch,
```

Comment:
so we don't support serveless for sigv4 anymore?

Reply:
Thanks for the comment~

We still support serverless.

This place refers to the initial state when we render sigv4, "es" will be by default selected.

Reply:
yup, it is the initial value.